packages/stream-tests/src/__tests__/model.test.ts

```diff
@@ -93,31 +101,35 @@ describe('Model API http-client tests', () => {
     expect(model.state.log.length).toEqual(2)
     expect(model.state.log[0].type).toEqual(CommitType.GENESIS)
     expect(model.state.log[1].type).toEqual(CommitType.ANCHOR)
-    expect(model.content).toEqual(MODEL_DEFINITION)
+    expect(JSON.stringify(model.content)).toEqual(JSON.stringify(modelContent))
   })
 
   test('Models are created deterministically', async () => {
-    const model1 = await Model.create(ceramic, MODEL_DEFINITION)
-    const model2 = await Model.create(ceramic, MODEL_DEFINITION)
+    const modelContent = modelContentWithName('DeterministicModel')
```

Comment:
Optional suggestion, you could use `expect.getState().currentTestName` to automatically get the test name and use it in the model name, so that you don't need to manually provide a different name for each model.  Could probably even do it within modelContentWithName directly and make it take no arguments.  Then would be nice to add a comment explaining why it matters that each test use a model with a different name.

Reply:
Another option, which I actually just did in a recent PR then undid at the last moment (see https://github.com/ceramicnetwork/js-ceramic/pull/2586#discussion_r1034495433) would be to create a new Ceramic instance for each test so that each test is totally independent of each other test (basically just means moving most of the beforeAll/afterAll into beforeEach/afterEach, for everything except IPFS setup/teardown).